src/python/WMCore/MicroService/Unified/MSOutput.py

```diff
@@ -109,9 +110,13 @@ def __init__(self, msConfig, mode, logger=None):
         self.msOutDB = MongoDB(**msOutDBConfig).msOutDB
         self.msOutRelValColl = self.msOutDB['msOutRelValColl']
         self.msOutNonRelValColl = self.msOutDB['msOutNonRelValColl']
-        self.ddm = DDM(url=self.msConfig['ddmUrl'],
-                       logger=self.logger,
-                       enableDataPlacement=self.msConfig['enableDataPlacement'])
+        if self.msConfig['defaultDataManSys'] == 'DDM':
```

Comment:
I agree with you and as we discussed yesterday on a chat this may happen only once this PR is merged: #9759

Because we are going to be using the same flag and the Rucio client will be instantiated on the master thread for both services: MSTransferror and MSOutput. Otherwise if I try to re-implement it here again, we will get into git conflicts that need to be resolved later and we are risking to loose one of the two PRs (depending which one is merged first), as it happened the last time.

Reply:
I hope I can finish that PR tomorrow, but if you want to not depend on that change, you can reuse (copy/paste) it from there and have it implemented here. Anyway is fine with me though.
Note that you will need to change the ms-output configuration with similar changes as I applied here:
https://github.com/dmwm/deployment/pull/924/files

Once you have a deployment PR, please link it from the description of this PR.

Reply:
I will deal with that later today.

Reply:
@amaltaro  are you quite far from finalizing this PR https://github.com/dmwm/WMCore/pull/9759 . I see we are not overlapping in files changed in our PRs and I would like to keep it like that. So It would be much cleaner if we merge your PR first and then I rebase  mine on top of that - this  way we will avoid eventual git conflicts, that would require manual resolution.

Reply:
Yes, I'd suggest to leave this one open and revisit it once I'm done with 9759, which is not going to be today.